### PR TITLE
New package: yeahwm 0.3.5

### PR DESCRIPTION
**Pull request by ananteris**

### CODE REVIEW

File: srcpkgs/yeahwm/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template build file for 'yeahwm'.
+pkgname=yeahwm
+version=0.3.5
+revision=1
+build_pie=no
```
Comment (xtraeme) [inline srcpkgs/yeahwm/template]: Setting `build_pie` has the same effect in all cases, if you don't want to apply hardened wrapper just remove this line.

File: srcpkgs/yeahwm/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+build_style=gnu-makefile
+make_build_args=""
```
Comment (xtraeme) [inline srcpkgs/yeahwm/template]: Empty variables are really useless, remove this.

File: srcpkgs/yeahwm/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+homepage="http://http://phrat.de/index.html"
+license="GPL-3"
+distfiles="http://phrat.de/${pkgname}_$version.tar.gz"
+checksum=c92ce98848c8b88d9b67c128e30710944c04a6454310254535c102b2b6fcee05
+
+do_build() {
+	make allinone
+}
+
+pkg_install() {
```
Comment (xtraeme) [inline srcpkgs/yeahwm/template]: `pkg_install` can only be used in `subpackages`, use `do_install()` instead.

File: srcpkgs/yeahwm/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Template build file for 'yeahwm'.
+pkgname=yeahwm
+version=0.3.5
+revision=1
+build_pie=no
+build_style=gnu-makefile
```
Comment (xtraeme) [inline srcpkgs/yeahwm/template]: Setting `build_style=gnu-makefile` to later override `do_build` and `do_install` is redundant, remove this.

File: srcpkgs/yeahwm/template [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+make_build_args=""
+makedepends="lesstif-devel libX11-devel libXinerama-devel"
+short_desc="A h* window manager for X based on evilwm and aewm"
+maintainer="Ananteris <ananteris@voidlinux.eu>"
```
Comment (xtraeme) [inline srcpkgs/yeahwm/template]: You need to be a committer to have a `voidlinux.eu` address, use another email address.

**Comment by xtraeme** [comment]
Those commit messages are uninformative... please check https://github.com/voidlinux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes to get more info about proper commit messages.

**Comment by xtraeme** [comment]
New packages and package updates should always set `revision=1`.

There's no need to bump it on any local change you make.

File: srcpkgs/yeahwm/template [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Template build file for 'yeahwm'.
+pkgname=yeahwm
+version=0.3.5
+revision=4
+makedepends="lesstif-devel libX11-devel libXinerama-devel"
+short_desc="A h* window manager for X based on evilwm and aewm"
+maintainer="Ananteris <ananteris@mailinator.com>"
+homepage="http://http://phrat.de/index.html"
```
Comment (diogoleal) [inline srcpkgs/yeahwm/template]: Change homepage for ```homepage="http://phrat.de"```

File: srcpkgs/yeahwm/template [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="http://phrat.de/${pkgname}_$version.tar.gz"
```
Comment (diogoleal) [inline srcpkgs/yeahwm/template]: in distfiles -> ```distfiles="http://phrat.de/${pkgname}_${version}.tar.gz"```

**Pull request closed without merging**